metrics/metrics.go

```diff
@@ -0,0 +1,80 @@
+package metrics
+
+import (
+	"github.com/go-kit/kit/metrics"
+	"github.com/go-kit/kit/metrics/multi"
+)
+
+// Registry has to implemented by any system that wants to monitor and expose metrics.
+type Registry interface {
+	ReqsCounter() metrics.Counter
+	ReqDurationHistogram() metrics.Histogram
+	RetriesCounter() metrics.Counter
+}
+
+// NewMultiRegistry creates a new multiRegistry that wraps multiple Registries.
+func NewMultiRegistry(registries []Registry) Registry {
+	reqsCounters := []metrics.Counter{}
+	reqDurationHistograms := []metrics.Histogram{}
+	retriesCounters := []metrics.Counter{}
+
+	for _, r := range registries {
+		reqsCounters = append(reqsCounters, r.ReqsCounter())
+		reqDurationHistograms = append(reqDurationHistograms, r.ReqDurationHistogram())
+		retriesCounters = append(retriesCounters, r.RetriesCounter())
+	}
+
+	return &multiRegistry{
+		reqsCounter:          multi.NewCounter(reqsCounters...),
+		reqDurationHistogram: multi.NewHistogram(reqDurationHistograms...),
+		retriesCounter:       multi.NewCounter(retriesCounters...),
+	}
+}
+
+// multiRegistry is a Registry that wraps multiple Registries and calls each of them when a Metric is tracked.
+type multiRegistry struct {
```

Comment:
Can you rename `multiRegistry` to `standardRegistry`?
This implementation is not specific to the multiple Registries.